src/scripting/lua_game.cc

```diff
@@ -78,7 +78,13 @@ Player
 
       local plr = wl.Game().players[1]                            -- the first player (usually blue)
       local plr = wl.Game().players[2]                            -- the second player
-      local plr = wl.Game().players[wl.Game().interactive_player] -- the interactive player
+      local i_plr
+      for p_idx, player in pairs(wl.Game().players) do
+         if player.number == wl.Game().interactive_player then
+            i_plr = player -- the interactive player
```

Comment:
Please try to indent the comment `-- the interactive player`  so it appears below the other comments.

Reply:
Agree, this will look better.